### PR TITLE
docs: Add ResourcePacks.md

### PR DESCRIPTION
**Pull request by spycrab**
Some documentation for resource packs, feel free to nitpick.

### CODE REVIEW

File: docs/ResourcePacks.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+```
+resource-pack.zip
+\__ manifest.json
+\__ logo.png (optional)
+\__ textures (Directory)
+    \__ GAMEID (Directory)
+        \___ TEXTURES GO HERE
+```
+
+**If you intend to use v1:** Your zip file may not be compressed or else **your pack will not load** (You can create uncompressed zips with software like 7-Zip).
```
[the remark this answers is not in the document]
Comment (spycrab) [reply docs/ResourcePacks.md]: [Doesn't seem to be needed.](https://ell.stackexchange.com/questions/73690/may-not-vs-must-not).

File: docs/ResourcePacks.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+**v2:** Supports compressed texture packs if ``compressed`` is ``true``.
+
+``GAMEID`` can be one or multiple directories which are named after:
+* a complete Game ID (e.g. ``SMNE01`` for "New Super Mario Bros. Wii (NTSC)")
+* one without a region (e.g. ``SMN`` for "New Super Mario Bros. Wii (All regions)").
+
+
+### logo.png
+
+The logo you provide has to be in ``PNG`` format and should not exceed the dimensions ``256x256`` pixels.
```
[the remark this answers is not in the document]
Comment (spycrab) [reply docs/ResourcePacks.md]: So ``Portable Network Graphics`` image format? I'll pass.

May change the dimensions stuff though.

File: docs/ResourcePacks.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+Resource packs are loaded from ``$DOLPHIN_USER_FOLDER/ResourcePacks``.
+
+## Removal
+
+Please remove the resource packs via the resource pack manager instead of just deleting them so they don't remain permanently activated.
+
+## Activation
+
+The actual installation process (*Activation*) just moves the contents of the pack's ``textures`` directory into ``$DOLPHIN_USER_FOLDER/Load/Textures``.
```
Comment (Zexaron) [inline docs/ResourcePacks.md]: It likely copies not moves right? I checked the ResourcePacks PR a bit, but I'm not sure.

File: docs/ResourcePacks.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Zexaron) [reply docs/ResourcePacks.md]: Yeah but probably nobody says `Portable Network Graphics` out loud or in their head when reading such abbreviations, I would suspect.

**Pull request merged**